### PR TITLE
Stop using Appveyor and Travis CI

### PR DESCRIPTION
**Pull request by rahulporuri**
This PR removes the appveyor configuration file and related utilities used by appveyor.
This PR also removes a number of scripts used to install edm on various platforms.

This PR is one step towards enthought/ets#63

**Checklist**
- [ ] ~Add a news fragment if this PR is news-worthy for end users. (see docs/releases/README.rst)~

### CODE REVIEW

**Comment by rahulporuri** [comment]
Note that I updated the branch protection settings in this repository to remove "Appveyor CI" as a required check and instead include the GitHub Actions CI checks as required.

File: .travis.yml [hunk 1]
```diff
@@ -1,61 +0,0 @@
...
-    - INSTALL_EDM_VERSION=3.0.1
-      PYTHONUNBUFFERED="1"
-
-matrix:
-  include:
-    - env: RUNTIME=3.6 TOOLKITS="null"
-    - env: RUNTIME=3.6 TOOLKITS="pyqt5 pyside2"
-    - env: RUNTIME=3.6 TOOLKITS="wx"
-  allow_failures:
-    - env: RUNTIME=3.6 TOOLKITS="wx"
```
Comment (aaronayres35) [inline .travis.yml:34]: Note testing on wx is now only done for the test-with-pip cron job.
Previously it was run every PR (but was an allowed failure so often failures went completely un-noticed anyway)

File: .travis.yml [hunk 2]
```diff
@@ -1,61 +0,0 @@
...
-  - mkdir -p "${HOME}/.cache/download"
-  - if [[ ${TRAVIS_OS_NAME} == 'linux' ]]; then ./install-edm-linux.sh; export PATH="${HOME}/edm/bin:${PATH}"; fi
-  - edm install -y wheel click coverage
-install:
-  - for toolkit in ${TOOLKITS}; do edm run -- python etstool.py install --runtime=${RUNTIME} --toolkit=${toolkit} || exit; done
-  - if [[ ${TRAVIS_EVENT_TYPE} == 'cron' ]]; then for toolkit in ${TOOLKITS}; do edm run -- python etstool.py install --runtime=${RUNTIME} --toolkit=${toolkit} --source || exit; done; fi
-  # etsdemo
-  - pushd ets-demo && for toolkit in ${TOOLKITS}; do edm run -- python etstool.py install --runtime=${RUNTIME} --toolkit=${toolkit} || exit; done && popd
-script:
-  - if [[ ${TOOLKITS} == 'null' ]]; then edm run -- python etstool.py flake8 --runtime=${RUNTIME} --toolkit=null; fi
```
Comment (aaronayres35) [inline .travis.yml:51]: we also now don't use the ci command `python etstool.py flake8` anywhere. However we still flake8 on PRs in the style-checks workflow with `python -m flake8`

**Review by aaronayres35 — APPROVED** [review]
LGTM

Just a couple non actionable comments

File: .travis.yml (hunk 2 above)
Comment (rahulporuri) [reply .travis.yml:51]: I think the `flake8` ci command is still useful for local development.

File: .travis.yml (hunk 1 above)
Comment (rahulporuri) [reply .travis.yml:34]: i'm okay with testing against wx only on cron jobs.

**Comment by rahulporuri** [comment]
I removed appveyor webhooks from this repository

**Pull request merged**